var/spack/repos/builtin/packages/qgis/package.py

```diff
@@ -161,7 +161,10 @@ def cmake_args(self):
                     self.spec['geos'].prefix.bin,
                     '-DGSL_CONFIG_PREFER_PATH=' + self.spec['gsl'].prefix.bin,
                     '-DPOSTGRES_CONFIG_PREFER_PATH=' +
-                    self.spec['postgresql'].prefix.bin
+                    self.spec['postgresql'].prefix.bin,
+                    '-DPROJ_INCLUDE_DIR=' + self.spec['proj'].prefix.include,
```

Comment:
I usually use:
```suggestion
                    '-DPROJ_INCLUDE_DIR=' + self.spec['proj'].headers.directories[0]
```
but that might not be strictly necessary.